proposals/20220825-huggingface-space-pusher.md

```diff
@@ -0,0 +1,85 @@
+#### SIG TFX-Addons
+# Project Proposal
+
+---
+
+**Your name:** Chansung Park
+
+**Your email:** deep.diver.csp@gmail.com
+
+**Your company/organization:** Individual ([ML GDE](https://developers.google.com/community/experts/directory/profile/profile-chansung-park))
+
+**Project name:** HuggingFace Space Pusher
+
+## Project Description
+HuggingFace Space Pusher(`HFSpacePusher`) pushes a model to the [HuggingFace Space Hub](https://huggingface.co/spaces).
```

Comment:
It might confuse users who're not familiar with Hugging Face Spaces. They may think that they're just taking a model and pushing it to a separate model repository which is clearly not the case. 

So, I think we need to be more specific about this description.